### PR TITLE
FOIA Fixes

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
A few of these are changes from comments in #606. The recent stuff are changes to fix the FOIA FAQ, form and request pages. Even though it'll be ripped out, it's easier to work with if it's fixed for now.

### Review
- @anselmbradford 
- @sebworks 

### Testing
- Visit /offices/foia-requests. All the subpages in the navigation should work.

### CODE REVIEW

**Comment by sebworks** [comment]
:+1:  I think we might be able to make the javascript a bit more general but that's out of scope for this PR.

**Comment by KimberlyMunoz** [comment]
Let's chat about it on Monday. I have another form on my plate that week,
so I could get it in then.

File: sub-pages/submit-a-request/index.html [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
     {% endif %}
 
-    {% if initiative.content %}
+    {% if sub_page.content %}
     <section class="initiative_description block block__flush-top">
-        {{ initiative.content | safe }}
+        {{ sub_page.content | safe }}
 
-        {# TODO: Extra Content. Remove once content updates. #}
+        <!-- Extra Content. Remove once content updates. -->
```
Comment (anselmbradford) [inline sub-pages/submit-a-request/index.html]: I'd keep this as jinja comments. Is there some reason this comment should appear in the HTML source output?

**Comment by anselmbradford** [comment]
One minor comment to fix as you see fit. Otherwise, :+1:

File: sub-pages/submit-a-request/index.html (hunk 1 above)
Comment (KimberlyMunoz) [inline sub-pages/submit-a-request/index.html]: Nope. Dunno how that snuck back in. 

Although that whole section is going away once we move to an actual template. Content needs to stick those steps into their content section.

**Pull request merged**